### PR TITLE
Add team_member API endpoint

### PR DESCRIPTION
**Pull request by munnsmunns**
Adds an API endpoint at `assessment/api/assessment/team_member/` that returns all Assessments where the user is a team member or project manager.

### CODE REVIEW

File: hawc/apps/assessment/api/viewsets.py [hunk 1]
```diff
@@ -271,6 +271,15 @@ def public(self, request):
         serializer = serializers.AssessmentSerializer(queryset, many=True)
         return Response(serializer.data)
 
+    @action(detail=False, permission_classes=(permissions.IsAuthenticated,))
+    def team_member(self, request):
+        """Get the list of assessments where the user is a team member or project manager."""
+        queryset = self.model.objects.filter(
+            Q(project_manager=request.user) | Q(team_members=request.user)
+        )
+        serializer = serializers.AssessmentSerializer(queryset, many=True)
```
Comment (munnsmunns) [inline hawc/apps/assessment/api/viewsets.py:280]: This serializer or the AssessmentMiniSerializer?

File: hawc/apps/assessment/api/viewsets.py [hunk 2]
```diff
@@ -271,6 +271,15 @@ def public(self, request):
         serializer = serializers.AssessmentSerializer(queryset, many=True)
         return Response(serializer.data)
 
+    @action(detail=False, permission_classes=(permissions.IsAuthenticated,))
+    def team_member(self, request):
+        """Get the list of assessments where the user is a team member or project manager."""
+        queryset = self.model.objects.filter(
+            Q(project_manager=request.user) | Q(team_members=request.user)
+        )
```
Comment (shapiromatron) [inline hawc/apps/assessment/api/viewsets.py:279]: add a distinct? sometimes the same person is added to both

File: hawc/apps/assessment/api/viewsets.py (hunk 1 above)
Comment (shapiromatron) [reply hawc/apps/assessment/api/viewsets.py:280]: this one is good I think, thanks!

**Review by shapiromatron — COMMENTED** [review]
Nice job, one minor tweak, should be good after that!

**Pull request merged**